docs/notebooks/verification_gridded_cosmo.ipynb

```diff
@@ -0,0 +1,3097 @@
+    "\n",
+    "For more info about the COSMO model see:\n",
+    "- https://www.cosmo-model.org/content/model/cosmo/coreDocumentation/cosmo_io_guide_6.00.pdf\n",
+    "- https://www.research-collection.ethz.ch/handle/20.500.11850/720460"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0311b50",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "from pathlib import Path\n",
+    "\n",
+    "import cartopy.crs as ccrs\n",
+    "import cartopy.feature as cfeature\n",
+    "import dask\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import xarray as xr\n",
+    "from dask.distributed import Client, LocalCluster\n",
+    "from pysteps.verification.salscores import sal  # requires scikit-image\n",
+    "from scipy.stats import wasserstein_distance, kurtosis, skew\n",
+    "from scores.continuous import (\n",
+    "    mae,\n",
+    "    mse,\n",
+    "    rmse,\n",
+    ")\n",
+    "from scores.continuous.correlation import pearsonr\n",
+    "from scores.spatial import fss_2d"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5a806b47",
+   "metadata": {},
+   "source": [
+    "**--------> Enter all your user settings in the cell below. <--------**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cf621a4c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### DEFAULTS ###\n",
+    "# This config will be applied to the data before any plotting. The data will be\n",
+    "# sliced and indexed according to the values in this config.\n",
+    "PATH_GROUND_TRUTH = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/cosmo.datastore.zarr\"\n",
+    "PATH_NWP = \"/capstor/store/cscs/swissai/a01/sadamov/cosmo_e_forecast.zarr\"\n",
+    "PATH_ML = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/eval_results/train-hi_lam-2x300-02_12_05-1462/test.zarr\"\n",
+    "PATH_BOUNDARY = \"/iopsstor/scratch/cscs/sadamov/pyprojects_data/neural-lam/era5_7_19_margin_interior.datastore.zarr\"\n",
+    "# elapsed forecast duration in steps for the forecast - [0] refers to the first forecast step\n",
+    "ELAPSED_FORECAST_DURATION = [0, 3, 6, 9, 12, 15, 18, 21]\n",
+    "# Select specific start_times for the forecast (will be used for all variables and metrics)\n",
+    "START_TIMES = [\"2020-02-13T00:00:00\", \"2020-02-15T00:00:00\"]\n",
+    "# Select specific plot times for the forecast (will be used to create maps for all variables)\n",
+    "PLOT_TIME = \"2020-02-13T00:00:00\"\n",
+    "# Selection spatial grid in projection\n",
+    "X = [None, None]\n",
+    "Y = [None, None]\n",
+    "# Map projection settings for plotting\n",
+    "PROJECTION = ccrs.RotatedPole(\n",
+    "    pole_longitude=190,\n",
+    "    pole_latitude=43,\n",
+    "    central_rotated_longitude=10,\n",
+    ")\n",
+    "# Define how variables map between different data sources\n",
```

Comment:
Would it be possible to annotate here which data sources each of these actually map between, in terms of PATH_GROUND_TRUTH, PATH_NWP, PATH_ML, PATH_BOUNDARY? Would help a lot if I will adapt this to DANRA later.

Reply:
I made this clearer. so basically `nwp, ground_truth, ml` all map to the same variable names for the interior using the dictionaries. Then this dictionary here maps the boundary to the interior variables.